### PR TITLE
initialName finding based on regex instead hardcoded line number

### PR DESCRIPTION
**Pull request by DaJansenGit**
This should fix the error described in #70. I don't know if it's a Dymola 2023 problem but it occurred for me with Dymola 2023.

solves #70

### CODE REVIEW

**Comment by DaJansenGit** [comment]
[dsin.txt](https://github.com/RWTH-EBC/ebcpy/files/12782621/dsin.txt)
[dsfinal.txt](https://github.com/RWTH-EBC/ebcpy/files/12782622/dsfinal.txt)

@FWuellhorst as requested I uploaded the dsin and dsfinal here for later testing adjustments

File: ebcpy/modelica/manipulate_ds.py [hunk 1]
```diff
@@ -37,17 +39,22 @@ def convert_ds_file_to_dataframe(filename):
...
 
     # Gets the X out of 'char initialName(X,Y)'
-    size_initial_names = int(content[number_line_initial_name].split("(")[-1].split(",")[0])
+    pattern_size_initial_name = r'^char initialName\((\d+),(\d+)\)$'
+    for number_line_initial_name, line in enumerate(content):
+        match_size_initial_name = re.match(pattern_size_initial_name, line)
+        if match_size_initial_name:
+            size_initial_names = int(match_size_initial_name.string.split("(")[-1].split(",")[0])
+            break
+
```
Comment (FWuellhorst) [inline ebcpy/modelica/manipulate_ds.py:53]: Maybe introduce an error of size_initial_names is still None?

```
    for number_line_initial_name, line in enumerate(content):
        match_size_initial_name = re.match(pattern_size_initial_name, line)
        if match_size_initial_name:
            size_initial_names = int(match_size_initial_name.string.split("(")[-1].split(",")[0])
            break
    else:
        raise ValueError("Could not find initial names in file")
```

**Review by FWuellhorst — COMMENTED** [review]
Thanks! I pushed the new test cases and increased the version number. Can you check my comment? Else, this looks good.  

Edit: Ok, i fucked up and pushed to the master branch :D

File: ebcpy/modelica/manipulate_ds.py (hunk 1 above)
Comment (DaJansenGit) [reply ebcpy/modelica/manipulate_ds.py:55]: good idea, added the valueError

**Review by FWuellhorst — APPROVED** [review]

**Pull request merged**